biostructbenchmark/main.py

```diff
@@ -0,0 +1,24 @@
+import argparse
+import os
+
+def check_file_exists(file_path: str):
```

Comment:
could we add a function to check that the file is a valid input type? i.e., can load a protein

likely need to choose a library to work with data